packages/internals/src/cli/getSchema.ts

```diff
@@ -361,41 +360,6 @@ function getRelativeSchemaPathSync(cwd: string): string | null {
   return null
 }
 
-/**
- * Sync version of the small helper that returns the directory which contains the `schema.prisma` file
- */
-export function getSchemaDirSync(schemaPathFromArgs?: string): string | null {
-  if (schemaPathFromArgs) {
-    return path.resolve(path.dirname(schemaPathFromArgs))
-  }
-
-  const schemaPath = getSchemaPathSync(schemaPathFromArgs)
-  if (schemaPath) {
-    return path.dirname(schemaPath)
-  }
-
-  return null
-}
-
-// TODO: This should probably return string | null to stay consistent with the other functions
-export function getSchemaSync(schemaPathFromArgs?: string): string {
```

Comment:
Note that this was used by third party generators / projects.
Since the async version exists, it might be fine in the end.